app/src/main/java/com/projectit3048c/MainActivity.kt

```diff
@@ -33,7 +33,7 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class MainActivity : ComponentActivity() {
     private var selectedFood: Food? = null
-    private val viewModel : MainViewModel by viewModel<MainViewModel>()
+    private val viewModel: MainViewModel by viewModel<MainViewModel>()
```

Comment:
There shouldn't be spaces before colons between a variable declaration and it's type.
https://kotlinlang.org/docs/coding-conventions.html#colon

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.